Terraform.sublime-syntax

```diff
@@ -597,6 +598,21 @@ contexts:
   #
   # https://github.com/hashicorp/hcl2/blob/master/hcl/hclsyntax/spec.md#structural-elements
   block:
+    # Special case heuristic for the AWS two-term resources
+    - match: (\b(resource)\b\s+(")(aws_{{identifier}})(")\s+(")({{identifier}})("))\s*(\{)
```

Comment:
Since `\s` is a word separator, `\b` is redundant.

```suggestion
    - match: (\b(resource)\s+(")(aws_{{identifier}})(")\s+(")({{identifier}})("))\s*(\{)
```